core/plugins/org.polarsys.capella.core.sirius.analysis/src/org/polarsys/capella/core/sirius/analysis/preferences/TitleBlockPreferencePage.java

```diff
@@ -66,7 +69,7 @@ public TitleBlockCell(String name, String content) {
 
     @Override
     public String toString() {
-      return name;
+      return name+"\n"+content; //$NON-NLS-1$

```

Comment:
Slight formatting issue ;) 
There are others on this file so a global format should do the job.